.eslintignore

```diff
@@ -3,7 +3,6 @@ coverage
 demos/aws-sdk
 demos/electron/*
 demos/browser/*.*
-demos/browser/app/meetingV2/*.*
 demos/browser/app/meetingReadinessChecker
```

Comment:
Do you mind fixing it for meetingReadinessChecker too :p

Reply:
I am planning to do that in a separate PR.